src/components/SearchBar.js

```diff
@@ -294,8 +294,12 @@ const DEFAULT_OPTIONS = [
   { value: "nairobi", label: "Nairobi, Kenya" },
   { value: "kisumu", label: "Kisumu, Kenya" },
   { value: "nakuru", label: "Nakuru, Kenya" },
-  { value: "lagos", label: "Lagos, Nigeria" },
   { value: "dar-es-salaam", label: "Dar-es-Salaam, Tanzania" },
+  { value: "abuja", label: "Abuja, Nigeria" },
+  { value: "lagos", label: "Lagos, Nigeria" },
+  { value: "port-harcourt", label: "Port Harcourt, Nigeria" },
+  { value: "ilorin", label: "Ilorin, Nigeria" },
+  { value: "maiduguri", label: "Maiduguri, Nigeria" },
```

Comment:
Sort cities in a given country alphabetically... And may be group them by country if it's [simple](https://react-select.com/advanced#replacing-builtins)